low-level/framework/utils/salt_util.py

```diff
@@ -14,83 +14,51 @@
 # cortx-questions@seagate.com.
 
 
-'''Module which provides common config information using Provisioner API'''
-
-
-import errno
-
-try:
-    from service_logging import logger
```

Comment:
Why we moved from logger to print?

Reply:
To avoid import failures when salt_util is called as a standalone python script from sspl_config